lib/cretonne/src/context.rs

```diff
@@ -54,7 +54,7 @@ impl Context {
     /// The `TargetIsa` argument is currently unused, but the verifier will soon be able to also
     /// check ISA-dependent constraints.
     pub fn verify<'a, ISA: Into<Option<&'a TargetIsa>>>(&self, _isa: ISA) -> verifier::Result {
-        verifier::verify_context(&self.func, &self.cfg, &self.domtree)
+        verifier::verify_context(&self.func, &self.cfg, &self.domtree, _isa.into())
```

Comment:
The leading underscore on `_isa` serves to silence Rust's dead code warning. I remove them when I start using an argument.